src/interpreter/Interpreter.as

```diff
@@ -665,8 +665,7 @@ public class Interpreter {
 
 	private function primReturn(b:Block):void {
 		// Return from the innermost procedure. If not in a procedure, stop the thread.
-		var didReturn:Boolean = activeThread.returnFromProcedure();
-		if (!didReturn) {
+		if (!activeThread.returnFromProcedure()) {
```

Comment:
I don't think this particular one was me but I sometimes do this type of thing so that I can inspect the value of a variable in a debugger. Thanks for the cleanup :)